src/txgraph.cpp

```diff
@@ -148,8 +148,9 @@ class Cluster
     /** Process elements from the front of args that apply to this cluster, and append Refs for the
      *  union of their descendants to output. */
     void GetDescendantRefs(const TxGraphImpl& graph, std::span<std::pair<Cluster*, DepGraphIndex>>& args, std::vector<TxGraph::Ref*>& output) noexcept;
-    /** Get a vector of Refs for all elements of this Cluster, in linearization order. */
-    void GetClusterRefs(TxGraphImpl& graph, std::span<TxGraph::Ref*> range, LinearizationIndex start_pos) noexcept;
+    /** Get a vector of Refs for all elements of this Cluster, in linearization order. Returns
```

Comment:
re: commit message for a0e19959e85b2ce1a90902ab1e3fe274f9d63a1d

Might just stress that this makes `m_excluded_cluster` a lot smaller since this takes the size from O(number cluster) to O(number non-singleton clusters).